test/podFailover/controller/cmd/main.go

```diff
@@ -35,9 +36,12 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/util/retry"
 	"k8s.io/klog/v2"
-	podfailure "sigs.k8s.io/azuredisk-csi-driver/test/podFailover/api"
+	podfailure "sigs.k8s.io/azuredisk-csi-driver/test/podFailover/apis/client/clientset/versioned"
+	podfailurev1beta1 "sigs.k8s.io/azuredisk-csi-driver/test/podFailover/apis/podfailure/v1beta1"
 )
 
+var kubeconfig = flag.String("kube-config", "", "kube config path for cluster")
```

Comment:
The standard parameter name for Kubernetes apps is `kubeconfig`. We should use it here and elsewhere.